datastore/google/cloud/datastore/batch.py

```diff
@@ -107,7 +107,7 @@ def namespace(self):
     def connection(self):
         """Getter for connection over which the batch will run.
 
-        :rtype: :class:`google.cloud.datastore.connection.Connection`
+        :rtype: :class:`google.cloud.datastore._http.Connection`
         :returns: The connection over which the batch will run.
         """
         return self._client.connection
```

Comment:
Nope. Will follow up with some more `connection`-privatization in another PR to keep the scope low here.